src/Morphir/TestCoverage/Backend.elm

```diff
@@ -0,0 +1,98 @@
+module Morphir.TestCoverage.Backend exposing (..)
+
+import AssocList as AssocDict
+import Dict exposing (Dict)
+import Json.Encode as Encode
+import Morphir.Correctness.BranchCoverage as BranchCoverage exposing (..)
+import Morphir.Correctness.Test exposing (TestCase, TestSuite)
+import Morphir.IR as IR exposing (..)
+import Morphir.IR.Distribution exposing (Distribution)
+import Morphir.IR.FQName as FQName exposing (FQName)
+import Morphir.IR.Module as Module exposing (ModuleName)
+import Morphir.IR.Name exposing (Name)
+import Morphir.IR.NodeId as NodeId exposing (..)
+import Morphir.IR.Package exposing (PackageName)
+import Morphir.IR.Value as Value
+
+
+type alias Coverage =
+    { numberOfBranches : Int
+    , numberOfCoveredBranches : Int
+    }
+
+
+type alias TestCoverageResult =
+    AssocDict.Dict NodeID Coverage
+
+
+createNodeId : FQName -> NodeID
+createNodeId fqn =
+    fqn |> ValueID
```

Comment:
This function is very simple and it looks like it's only used once so it would be cleaner to just inline it.